website/src/libs/api-v3/firebase.js

```diff
@@ -0,0 +1,68 @@
+import Firebase from 'firebase';
+import nconf from 'nconf';
+const IS_PROD = nconf.get('IS_PROD');
+const FIREBASE_CONFIG = nconf.get('FIREBASE');
+const FIREBASE_ENABLED = IS_PROD && FIREBASE_CONFIG.ENABLED === 'true';
+
+let firebaseRef;
```

Comment:
yeah but in this case it's not a normal instance, but a reference to a specific path in firebase